packages/gestalt/src/Module.js

```diff
@@ -1,33 +1,39 @@
 // @flow strict
 import React, { type Node } from 'react';
 import Box from './Box.js';
-import { type BaseProps } from './moduleTypes.js';
+import Flex from './Flex.js';
 import ModuleTitle from './ModuleTitle.js';
 import ModuleExpandable from './ModuleExpandable.js';
+import { type BaseModuleProps } from './moduleTypes.js';
+
+type Props = {|
+  ...BaseModuleProps,
+  id: string,
+|};
 
 export default function Module({
   children,
-  id,
   icon,
   iconAccessibilityLabel,
+  id,
   title,
   type = 'info',
-}: BaseProps): Node {
+}: Props): Node {
   return (
-    <Box id={id} rounding={2} borderStyle="shadow" direction="column">
-      {title && (
-        <Box padding={6} display="flex">
+    <Box borderStyle="shadow" id={id} padding={6} rounding={4}>
```

Comment:
Main change

Reply:
Yep, and the same on ModuleExpandable. Sorry, meant to call that out 😅